plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -108,7 +108,7 @@ function() {
 					echo '<div class="error"><p>';
 					printf(
 						/* translators: %s: is referring to the plugin's name. */
-						esc_html__( '%1$s plugin has been deactivated to avoid conflicts with %2$s plugin.', 'woocommerce' ),
+						esc_html__( '%1$s plugin has been deactivated as %1$s plugin is now included with %2$s plugin.', 'woocommerce' ),
```

Comment:
```suggestion
						esc_html__( The '%1$s plugin has been deactivated to avoid conflicts with newer admin features in the %2$s plugin.', 'woocommerce' ),
```